### DISCUSSION TITLE
GitHub Team Purchase Plan - Reg

### DISCUSSION THREAD

**Discussion by NARAIN2803** [community/community#72200] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

We are in the process of considering the purchase of a GitHub Team plan.  The possibility of restricting access to only our organization's email addresses and preventing requests from being sent to personal email addresses from our GitHub account.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @NARAIN2803! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, do you have a specific question here? This was written moreso as a statement so if you do have a question or are sharing feedback, can you clarify that a bit? 

I've gone ahead and moved this to the correct category for you. Good luck!
